src/Directory.Build.props

```diff
@@ -10,6 +10,7 @@
   <Import Project="$(RepoToolsetDir)Settings.props" />
 
   <PropertyGroup>
+    <StrongNameKeyId>MicrosoftAspNet</StrongNameKeyId>
```

Comment:
Won't changing the key break things?

Reply:
The key was already changed, incorrectly.

This is putting it back to the key we shipped with